portfolio/src/main/webapp/script.js

```diff
@@ -26,8 +24,12 @@ function autoChange (){
 setInterval(autoChange, 5000);
 
 function loadComments() {
-    fetch("/data").then(response => response.json()).then((commentList) => {
-        console.log(commentList);
+    let root = document.getElementById("history");
+    while(root.firstChild){root.removeChild(root.firstChild);}
```

Comment:
Luckily we can do this more simply with `.empty()` : )
https://www.w3schools.com/jquery/html_empty.asp

Reply:
Ah shoot, my bad, I completely missed that that's a jquery thing (we use a similar library internally with a similar method). Instead with vanilla JS you'd want to use `.innerHTML = '';`, more context at https://stackoverflow.com/questions/3955229/remove-all-child-elements-of-a-dom-node-in-javascript

Reply:
Yah I noticed that. I added in jQurey to make it work, but if you prefer the method you just suggest I can do that as well.

Reply:
Yeah let's stick with vanilla JS, sorry about the mixup on my part